File: prime number/PrimeNumber.php
<?php
function isPrimeNumber(int $number) : bool {
    if ($number <= 1) {
        return false;
    }

    for ($i=2; $i <= sqrt($number) ; $i++) {
        if ($number % $i == 0) {
            return false;
        }
    }

    return true;
}
for ($i=1; $i <= 10; $i++) {
    if (isPrimeNumber($i)){
        echo $i . "\n";
    }

}

?>

File: prime number/PrimeNumber.py
import math

def isPrimeNumber(number : int) -> bool:
    if number <= 1:
        return False

    for i in range(2, int(math.sqrt(number)) + 1):
        if number % i == 0:
            return False

    return True

for i in range(1, 10):
    if isPrimeNumber(i):
        print(i)
